requirements/requirements.txt

```diff
@@ -1,10 +1,9 @@
-Flask==0.10.1
-Flask-Environments==0.1
```

Comment:
Did you mean to remove `Flask-Environments` from the requirements?  I am getting an `ImportError` when trying to start the application as it can't find the module.